src/Microsoft.Azure.Functions.Extensions.Mcp/McpToolExtensions.cs

```diff
@@ -35,11 +35,18 @@ internal static JsonElement GetPropertiesInputSchema(this IMcpTool tool)
                     writer.WritePropertyName("items");
                     writer.WriteStartObject();
                     writer.WriteString("type", p.PropertyType);
+
+                    // Add enum values for array items if available
+                    WriteEnumValuesIfPresent(writer, p.EnumValues);
+
                     writer.WriteEndObject();
                 }
                 else
                 {
                     writer.WriteString("type", p.PropertyType);
+
+                    // Add enum values for scalar properties if available
+                    WriteEnumValuesIfPresent(writer, p.EnumValues);
```

Comment:
Do you need to have this in each if-branch? Could this live outside of this if-else since the code is the same regardless?